cmd/gossamer/configcmd.go

```diff
@@ -85,10 +88,10 @@ func makeNode(ctx *cli.Context) (*dot.Dot, *cfg.Config, error) {
 		return nil, nil, err
 	}
 
-	log.Info("🕸\t Configuring node...", "datadir", currentConfig.Global.DataDir, "protocol", currentConfig.P2p.ProtocolID, "bootnodes", currentConfig.P2p.Bootnodes)
+	// TODO: check Roles to see if we need p2p service (Roles 0 = no network)
```

Comment:
#601

Reply:
would be better if you could add the raised ticket to the TODO, so it allows good link between the code and the reported issue:
```
// TODO: check Roles to see if we need p2p service (Roles 0 = no network) #601 
```